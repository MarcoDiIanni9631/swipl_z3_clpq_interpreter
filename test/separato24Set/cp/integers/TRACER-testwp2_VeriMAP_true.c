# 1 "MAP/SAFE-exbench/TRACER-testwp2.tmp.c"
# 1 "<command-line>"
# 1 "MAP/SAFE-exbench/TRACER-testwp2.tmp.c"
# 23 "MAP/SAFE-exbench/TRACER-testwp2.tmp.c"
main()
{
  int x, y, z;
  if (z>0) {

    x = 1;
  } else {
    x = 2;
  }

  if (x<0) {

    y = -1;
  } else {

    y = 1;
  }

  __VERIFIER_assert(!( y<0 ));
}
