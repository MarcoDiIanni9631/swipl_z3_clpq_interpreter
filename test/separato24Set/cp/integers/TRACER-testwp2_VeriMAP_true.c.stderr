MAP/SAFE-exbench/TRACER-testwp2.tmp.c:41: Warning: Body of function main falls-through. Adding a return statement
